### ISSUE TITLE
[1.4.x] Tabs disappear when clicked

### ISSUE THREAD

**Issue by espaan** [zikula/core#2238]
Just tried Content (4.2.0) so old style module etc with 1.4.x. Most things seem to work perfectly.
Of course layouts can be converted into col-md-x columns instead of the "hard coded" 62-38/etc% column divisions, but that is a small thing.

However the Tabs that are using {formtabbedpanel __title='Content'} is not all ok. There are 3 tabs, when you click one the 2nd the first dissappears, when you click on the 3rd the 2nd also dissappears, so in the end you're left with only the 3rd tab. 

The tabs is hidden via jQuery call somehow it seems. Not checked in details where it could come from.

**Comment by craigh** [comment]
I noticed this in EL also.

**Comment by espaan** [comment]
Weird, somehow the tab as well as the content div is hidden instead of only the content div not visible.

**Comment by craigh** [comment]
@phaidon - could this have something to do with some of the "magic" you added to help jQuery and bootstrap play well together?

**Comment by cmfcmf** [comment]
Probably related to https://github.com/zikula/core/issues/1190.

**Comment by craigh** [comment]
refs https://github.com/zikula/core/commit/079df47e7c1f536a0d9eea2993ae19768e1f0554

**Comment by cmfcmf** [comment]
Random guess: It does not work because jQuery is loaded after prototype:
> <script type="text/javascript" src="http://zikula.org/javascript/ajax/proto_scriptaculous.combined.min.js"></script>
<script type="text/javascript" src="http://zikula.org/web/jquery/jquery.min.js"></script>

**Comment by espaan** [comment]
And it should :-) http://api.jquery.com/jquery.noconflict/#jQuery-noConflict-removeAll
the noconflict is always adviced to load prototype first and then jQ

**Comment by craigh** [comment]
no it doesn't work.

**Comment by craigh** [comment]
> And it should :-) 

but the "fix" checks for jQuery - and now this happens BEFORE jQuery exists.

**Comment by craigh** [comment]
we need the sorcerer. ping @phaidon PLEASE HELP!

**Comment by espaan** [comment]
Ah right, that makes it more difficult indeed. mmhh. Then jQuery should be loaded first for the fix, but is adviced to load after prototype in noconflict description..

**Comment by craigh** [comment]
I wonder if the fix could be extracted from it's current location and loaded later.

**Comment by espaan** [comment]
In my case jQuery is loaded before prototype, see below. But still the tabs dissappear.

```
<script type="text/javascript" src="/zk140/web/jquery/jquery.js"></script>
<script type="text/javascript" src="/zk140/javascript/jquery_config.js"></script>
<script type="text/javascript" src="/zk140/web/jquery/jquery-migrate.min.js"></script>
<script type="text/javascript" src="/zk140/web/jquery-ui/jquery-ui.js"></script>
<script type="text/javascript" src="/zk140/javascript/ajax/original_uncompressed/prototype.js"></script>
<script type="text/javascript" src="/zk140/javascript/ajax/original_uncompressed/effects.js"></script>
<script type="text/javascript" src="/zk140/javascript/ajax/original_uncompressed/builder.js"></script>
<script type="text/javascript" src="/zk140/javascript/ajax/original_uncompressed/controls.js"></script>
<script type="text/javascript" src="/zk140/javascript/ajax/original_uncompressed/dragdrop.js"></script>
<script type="text/javascript" src="/zk140/javascript/ajax/original_uncompressed/slider.js"></script>
<script type="text/javascript" src="/zk140/javascript/ajax/original_uncompressed/sound.js"></script>
<script type="text/javascript" src="/zk140/javascript/helpers/Zikula.js"></script>
...
```

**Comment by craigh** [comment]
I think this is the fix but am sort of clueless on implementation

http://softec.lu/site/DevelopersCorner/BootstrapPrototypeConflict

**Comment by espaan** [comment]
It should be
```
  jQuery('*').on('hide.bs.tab', function( event ) {
    isBootstrapEvent = true;
  });
```
and not hide.bs.tabs in the prototype fix :-) then it works in my case.

**Comment by craigh** [comment]
> and not hide.bs.tabs

I don't see `tabs` plural in the fix?

**Comment by espaan** [comment]
No sorry, my bad. You had hide.bs.tab. Updated comment. We were only missing hide.bs.tab is seems in the prototype fix.

**Comment by espaan** [comment]
So also update line 514 in https://github.com/zikula/core/blob/1.4/src/lib/util/JCSSUtil.php#L514
and remove prototype "hack"
JCSSUtil line 514 should be updated 
```
            $scripts = array_merge($jQueryUncompressed, $jQueryUiUncompressed, $prototypeUncompressed, $livepipeUncompressed, array_slice($scripts, 5));
```
into
```
			$scripts = array_merge($prototypeUncompressed, $jQueryUncompressed, $jQueryUiUncompressed, $livepipeUncompressed, array_slice($scripts, 5));
```

**Comment by craigh** [comment]
closed by #2242
